### ISSUE TITLE
Restrict blocks in patterns

### ISSUE THREAD

**Issue by reasterbrooknc** [utkwdn/utkwds#235]
Hi there @pikemurdy—there's a newish option to restrict allowed blocks within patterns. @nc-mhenderson and I think this could be useful for our component library and wanted to get your thoughts. 

Details here: https://developer.wordpress.org/news/2024/01/29/how-to-disable-specific-blocks-in-wordpress/#disable-blocks-using-the-allowedblocks-attribute

**Comment by pikemurdy** [comment]
Yes we disabled a bunch of blocks back when we were rolling our own theme. I think there are some blocks that are not useful and confusing ("verse" comes to mind). But I'm not 100% sure how to identify exactly which ones to restrict.

Digging around that link, there are ways to [disable styles](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-styles/) as well. We'll probably want to eventually dig into that. (I'm sure we'll never want anyone creating duotone images on the fly).

Is there any word on disabling the the block settings? Id like there to be a better solution than just hiding it with CSS.
